python/rmm/CMakeLists.txt

```diff
@@ -27,7 +27,16 @@ include(rapids-cython-core)
 rapids_cython_init()
 
 # pass through logging level to spdlog
-add_compile_definitions("SPDLOG_ACTIVE_LEVEL=SPDLOG_LEVEL_${RMM_LOGGING_LEVEL}")
+add_compile_definitions("RMM_LOG_ACTIVE_LEVEL=RMM_LOG_LEVEL_${RMM_LOGGING_LEVEL}")
+
+# Create an object library for the logger so that we don't have to recompile it. This assumes that
+# we always have logging enabled when we built the C++ library since the targets that are created
+# when we find_package(rmm) already have that setting baked in. TODO: Figure out how to use
+# define_property in the C++ build to make SUPPORTS_LOGGING a queryable property that can be checked
+# here. Otherwise we just assume that the Python package is never built against a C++ rmm without
+# logging support.
+add_library(cpp_logger OBJECT)
+target_link_libraries(cpp_logger PRIVATE rmm::rmm_logger_impl)
```

Comment:
Should we support building the Python package against a C++ rmm without logging, or can we just assume that logging is always supported? I really only added that in the C++ layer for someone who really needs to use rmm in header-only mode and doesn't want to introduce extra TUs.

Reply:
No longer relevant.